azurelinuxagent/ga/policy/policy_engine.py

```diff
@@ -51,6 +45,16 @@ def __init__(self, msg, inner=None):
         super(InvalidPolicyError, self).__init__(msg, inner)
 
 
+class ExtensionPolicyError(ExtensionError):
+    """
+    Error raised during agent extension policy enforcement.
+    """
+    # TODO: when CRP adds terminal error code for policy-related extension failures, set that as the default code.
+    def __init__(self, msg, inner=None, code=-1):
```

Comment:
should this default code be 1009 (terminal enable failure) for now? or just remove the default?

Reply:
I think you can remove the default